src/CMakeLists.txt

```diff
@@ -57,14 +57,13 @@ add_library(migraphx
     json.cpp
     load_save.cpp
     make_op.cpp
+    memory_coloring.cpp
     module.cpp
     msgpack.cpp
     normalize_attributes.cpp
     normalize_ops.cpp
     op_enums.cpp
     operation.cpp
-    opt/memory_coloring.cpp
-    opt/memory_coloring_impl.cpp
```

Comment:
These files needs to be deleted/removed.

Reply:
Yes, I plan to remove later to avoid merge conflicts.